Sources/Foundation/FileManager.swift

```diff
@@ -25,6 +25,10 @@ import WinSDK
 import WASILibc
 #endif
 
+#if canImport(Android)
```

Comment:
Since this is mutually exclusive with Windows and WASI above, mind turning it into one block of `#elseif os(WASI)... #elseif canImport(Android)`?